int64[2] data